packages/cli/tests/e2e/commonUtils.ts

```diff
@@ -94,6 +94,10 @@ export function getUniqueAppName() {
   return getAppNamePrefix() + Date.now().toString() + uuidv4().slice(0, 2);
 }
 
+export function convertToAlphanumericOnly(appName: string): string {
+  return appName.replace(/[^\da-zA-Z]/g, "");
```

Comment:
Just wondering will this happen: the appname contains only special characters, and after replacing it becomes an empty string.

Reply:
Never mind, it should start from a letter.